### ISSUE TITLE
Default maximum response size to 2MB is not working

### ISSUE THREAD

**Issue by umeshtiwari23** [akamai/esi-test-server-docker#13]
Release Notes
ESI Test Server
Version 6.0.0.6x
April 17, 2017
*********************************************************************
- Expand default maximum response size to 2MB;

I am using the version 6.0.0.9 and I can see that If my page size increases 1MB, ETS start giving the 500 error. If the page size is lower than 1 MB then it is working fine

I tried to add the command --page_size_limit 4000000 but it has no effect as well. Please, can someone let me know how to set this limit or it is something not fixed in the latest release?

I am using akamaiesi/ets-docker:6.0.0.9 with command --page_size_limit 4000000
My page size is 1033701 and still fails as it is exceeding 1MB

**Comment by tdickers** [comment]
You are correct. This appears to be a valid bug report. I'll open a ticket internally and raise this to the team.

[This gist](https://gist.github.com/tdickers/481dc3f761b2b1131730444aa1b6738a) (download as zip) contains a reproduction.

When I run with `--debug origin` I see 
```
<Eval>
Error: [2010]: Your code is too big.  Your total file size must be less than 1000000 bytes.
[2016]: Your ESI source file is too large to display the line numbers with the source. Only ESI source code less than 3000000 bytes will show line.numbers in the debug output.
Error: [2005]: Your code did not run successfully.
</Eval>
```

**Comment by umeshtiwari23** [comment]
@tdickers Do you know a timeline when this defect will be fixed?

**Comment by tdickers** [comment]
I'm not able to offer an ETA at this time, but you may find the Akamai Sandbox provides better compatibility with Akamai Production and Staging. Any ESI page size limit increases that have been applied to your properties should be usable there. Please see https://developer.akamai.com/tools/akamai-sandbox for more information.

**Comment by jmaidment** [comment]
Hi @tdickers, is there any update on this? Is there a ticket reference that you're able to share?

**Comment by jcamelon** [comment]
@tdickers is no longer part of Akamai.   
Her previous post is still valid in that ESI test server is a means of approximating ESI behaviour within a container for localized testing.  It has certain flaws which are not representative on the Akamai edge network, and generally we are encouraging people to use the akamai sandbox for a more up to date and supported workflow.  

This is not to say that we will not be periodically updating ETS with new versions ... however, the frequency may be reduced.

**Comment by jmaidment** [comment]
Thanks @jcamelon , appreciate the reply
